src/components/DropHover.tsx

```diff
@@ -18,10 +18,12 @@ const DropHover = ({
   isHovering,
   prevChildId,
   simplePath,
+  isTableCol1,
 }: {
   isHovering: boolean
   prevChildId?: ThoughtId
   simplePath: SimplePath
+  isTableCol1?: boolean
```

Comment:
By adding `isTableCol1` as a prop, it adds dependencies to four different components as it gets passed down the hierarchy:

LayoutTree → VirtualThought → Subthought → Thought → DropHover

While this is performant, the amount of components it touches is not ideal. Instead, we can subscribe to the selector directly in the DropHover component. It's a fast selector, so the additional computation is negligible:

```ts
  const isTableCol1 = useSelector(state =>
    attributeEquals(state, head(rootedParentOf(state, simplePath)), '=view', 'Table')
  )
```

Reply:
Thanks for this, I wasn't also too sure about drilling prop into so many components but this solved it.